autospec/specfiles.py

```diff
@@ -328,7 +328,10 @@ def write_files(self):
                 continue
 
             self._write("\n%files {}\n".format(pkg))
-            self._write("%defattr(-,root,root,-)\n")
+            if pkg in ["doc"]:
```

Comment:
Why not pkg == "doc"?

Reply:
In case we want to add the debug info shortly

Reply:
I'll look forward to it.